src/derivatives/atlas.md

```diff
@@ -472,6 +473,141 @@ A guide for using macros can be found at
 })
 }}
 
+**Producing a new template AND atlas.**
+Atlasing is often performed with reference to a *custom* standard space.
+In this case, a feature template map is generated from all the participant(s)
+in the study, and the atlas' artifacts are produced with reference to that
```

Comment:
It's not clear what artifacts means? Should it be label instead?
Could also just say "and the atlas is produced with reference..."

Reply:
Responded below, because it can mean several things (segmentations, parcellations, landmarks, connected landmarks, fiber bundle geometries, etc.), I used the term *artifacts*.

Happy to find a good alternative if it is confusing (e.g., with imaging *artifacts*).